app/views/static_pages/contributing.html.erb

```diff
@@ -12,48 +12,42 @@
         </div>
       </div>
 
+      <%
```

Comment:
It wasn't mentioned, but for me it was easier to work this way. And didn't reversed it, because it seemed to be useful for later.